Source/Core/ArcGisImageServerTerrainProvider.js

```diff
@@ -61,6 +60,7 @@ define([
      *   proxy : new Cesium.DefaultProxy('/terrain/')
      * });
      * viewer.terrainProvider = terrainProvider;
+     *  @see TerrainProvider
```

Comment:
Can you please include a line of whitespace between the end of the `@example` code and the `@see` tag, e.g.,
```
    * viewer.terrainProvider = terrainProvider;
    *
    * @see TerrainProvider
```